src/app/pages/account-payment/account-payment/account-payment.component.html

```diff
@@ -46,6 +46,8 @@ <h4>
       </a>
     </div>
     <p>{{ pi.accountIdentifier }}</p>
-    <a *ngIf="!isPreferred" (click)="setAsDefaultPayment(pi.id)">{{ 'account.payment.preferred.link' | translate }}</a>
+    <a *ngIf="!isPreferred" (click)="setAsDefaultPayment(pi.id)">{{ 'account.payment.preferred.link' | translate }}</a
+    >&nbsp;&nbsp;&nbsp;
+    <ish-account-payment-concardis-directdebit [paymentInstrument]="pi"></ish-account-payment-concardis-directdebit>
```

Comment:
ui should be improved, the link should be shown in a new line without any left padding

Reply:
Done.